Sources/SwifterSwift/Combine/PublisherExtensions.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  PublisherExtensions.swift
+//  SwifterSwift
+//
+//  Created by Heecheon Park on 7/28/23.
+//  Copyright © 2023 SwifterSwift
+//
+
+// swiftlint:disable large_tuple
```

Comment:
Don't disable for the whole file, only the function where you need it.

Reply:
@heecheon92 just move this down into extension , let's say line 15, and the enabling up to 120.

Reply:
fixed at bf46972a2f85cb9ffc39518a74154b24f76c5793